ocaml/xapi/cli_operations.ml

```diff
@@ -2040,12 +2040,25 @@ let vm_memory_target_wait printer rpc session_id params =
 			let vm=vm.getref () in
 			Client.VM.wait_memory_target_live rpc session_id vm) params [])
 
-let vm_call_plugin printer rpc session_id params =
+let vm_call_plugin fd printer rpc session_id params =
 	let vm_uuid = List.assoc "vm-uuid" params in
 	let vm = Client.VM.get_by_uuid rpc session_id vm_uuid in
 	let plugin = List.assoc "plugin" params in
 	let fn = List.assoc "fn" params in
 	let args = read_map_params "args" params in
+	(* Syntax interpretation: args:key:file=filename equals args:key=filename_content *)
+	let convert ((k,v) as p) =
+		match String.split ~limit:2 ':' k with
+		| key :: annot :: [] when annot = "file" ->
```

Comment:
Since the `annot` symbol isn't used anywhere else, why not just match against `key :: "file" :: []` ?